examples/user_interfaces/combine_with_vtk_sgskip.py

```diff
@@ -0,0 +1,142 @@
+"""====================================
+Combining with VTK in a Qt interface
+====================================
+
+`VTK <https://www.vtk.org/>`_ is a library for 3D visualization.
+
+This example shows how to embed an interactive Matplotlib figure along
+side a VTK renderer in a Qt UI.
+
+TODO:
+
+ - use more meaningful example data
+ - change cylinder to isosurface view of data
+
+"""
+
+
+from __future__ import print_function
+
+import sys
+
+import numpy as np
+from matplotlib.figure import Figure
+from matplotlib.backends.backend_qt5agg import (
+    FigureCanvasQTAgg as FigureCanvas,
+    NavigationToolbar2QT as NavigationToolbar)
+
+from PyQt5 import QtWidgets, QtCore
+
+import vtk
+
+from vtk.qt.QVTKRenderWindowInteractor import QVTKRenderWindowInteractor
+
+
+def make_vtk_plane(ren):
+
+    # create source
+    source = vtk.vtkPlaneSource()
+    source.SetPoint1(-10, 10, 0)
+    source.SetPoint2(10, -10, 0)
+    source.SetOrigin(-10, -10, 0)
+    # mapper
+    mapper = vtk.vtkPolyDataMapper()
+    mapper.SetInputConnection(source.GetOutputPort())
+
+    # actor
+    actor = vtk.vtkActor()
+    actor.SetMapper(mapper)
+
+    # assign actor to the renderer
+    ren.AddActor(actor)
+
+    return source, mapper, actor
```

Comment:
I would only return the `actor`, the end of the pipeline. The rest is static setup and should not be needed in principle.